lib/chart/animated_line_chart.dart

```diff
@@ -588,6 +631,57 @@ class ChartPainter extends CustomPainter {
     }
   }
 
+  void _drawShadedAreaBetweenLines(Size size, Canvas canvas) {
+    assert(filledMarkerLinesValues!.length ==
+        _chart.lines.where((line) => line.isMarkerLine).length);
+
+    List values = [];
+
+    if (_chart.seriesMap != null) {
+      _chart.seriesMap!.forEach((key, value) {
+        if (key == 0) {
```

Comment:
Reason why is it skipped ?

Reply:
Yes, the key at index 1 is the line with all the measurements and we don't want to have any shaded areas starting from that line as it is not a markerline